flake.nix

```
@@ -0,0 +1,31 @@
+{
+  description = "Sourcegraph CLI";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+      in
+      rec {
+        packages.src-cli = with pkgs; buildGoModule {
```

```suggestion
        packages.src-cli = with pkgs; buildGoModule.override { go = pkgs.go_1_19; } {
```

Just so we're consistently using 1.19